ownCloudTests/SettingsTests.swift

```diff
@@ -83,20 +109,31 @@ class SettingsTests: XCTestCase {
 			.usingSearch(grey_scrollInDirection(GREYDirection.down, 100), onElementWith: grey_text("Log file".localized))
 			.assert(grey_sufficientlyVisible())
 		EarlGrey.select(elementWithMatcher: grey_accessibilityID("Logging"))
-			.usingSearch(grey_scrollInDirection(GREYDirection.down, 100), onElementWith: grey_accessibilityID("share-logfile"))
-			.assert(grey_sufficientlyVisible())
-		EarlGrey.select(elementWithMatcher: grey_accessibilityID("Logging"))
-			.usingSearch(grey_scrollInDirection(GREYDirection.down, 100), onElementWith: grey_accessibilityID("reset-logfile"))
-			.assert(grey_sufficientlyVisible())
-		EarlGrey.select(elementWithMatcher: grey_accessibilityID("Logging"))
-			.usingSearch(grey_scrollInDirection(GREYDirection.down, 100), onElementWith: grey_accessibilityID("mask_private_data"))
+			.usingSearch(grey_scrollInDirection(GREYDirection.down, 100), onElementWith: grey_text("Browse".localized))
 			.assert(grey_sufficientlyVisible())
 
 		//Reset status
 		EarlGrey.select(elementWithMatcher: grey_text("Settings".localized)).perform(grey_tap())
 		EarlGrey.select(elementWithMatcher: grey_text("ownCloud".localized)).perform(grey_tap())
 	}
 
+	/*
+	* PASSED if: Log level is changed to "Warning".
+	*/
+	func testSwitchLogLevel () {
+
+		//Actions
+		EarlGrey.select(elementWithMatcher: grey_accessibilityID("logging")).perform(grey_tap())
+		EarlGrey.select(elementWithMatcher: grey_text("Warning".localized)).perform(grey_tap())
+		EarlGrey.select(elementWithMatcher: grey_text("Settings".localized)).perform(grey_tap())
+
+		//Assert
+		EarlGrey.select(elementWithMatcher: grey_text("Warning".localized)).assert(grey_sufficientlyVisible())
+
+		//Reset status
+		EarlGrey.select(elementWithMatcher: grey_text("ownCloud".localized)).perform(grey_tap())
```

Comment:
Please could you replace "ownCloud" with `OCAppIdentity.shared.appName`?

Reply:
👍